### PR TITLE
alphabetise, deselect button and scale

### PR DESCRIPTION
**Pull request by BexDunn**
- alphabetised catchment list
- created a deselect button
- set scale as a parameter in a function for bubble plots

### CODE REVIEW

File: examples/inundation_qld.ipynb [hunk 1]
```diff
@@ -9,30 +24,65 @@
...
     "from bokeh.io import curdoc, output_notebook, show\n",
     "from bokeh.layouts import layout, column, row, WidgetBox, gridplot\n",
-    "from bokeh.models import CheckboxGroup, Select,  CategoricalColorMapper, ColumnDataSource,HoverTool, Label, SingleIntervalTicker, Slider, DatetimeTickFormatter, YearsTicker, Legend, TapTool, CustomJS, LegendItem, field\n",
+    "from bokeh.models import Button, CheckboxGroup, Select,  CategoricalColorMapper, ColumnDataSource,HoverTool, Label, SingleIntervalTicker, Slider, DatetimeTickFormatter, YearsTicker, Legend, TapTool, CustomJS, LegendItem, field\n",
     "from bokeh.palettes import plasma\n",
     "from bokeh.plotting import figure\n",
     "from bokeh.transform import factor_cmap\n",
     "from bokeh.events import DoubleTap\n",
-    "import os, sys, urllib\n",
+    "import os, sys, urllib"
```
Comment (emmaai) [inline examples/inundation_qld.ipynb]: this line is duplicate

File: examples/inundation_qld.ipynb [hunk 2]
```diff
@@ -9,30 +24,65 @@
     "import fiona\n",
     "import numpy as np\n",
     "import pandas as pd\n",
+    "import os, sys, urllib\n",
+    "\n",
+    "#add path to module so that we can re-run the notebook\n",
+    "sys.path.append('../../wit_tooling/')\n",
```
Comment (emmaai) [inline examples/inundation_qld.ipynb]: please don't do this we could do something like `!pip install git+git://github.com/GeoscienceAustralia/wit_tooling` at the very beginning of the notebook if this notebook is not distributed with the package

**Review by emmaai — CHANGES_REQUESTED** [review]

File: examples/inundation_qld.ipynb (hunk 2 above)
Comment (BexDunn) [reply examples/inundation_qld.ipynb]: thanks! done

File: examples/inundation_qld.ipynb (hunk 1 above)
Comment (BexDunn) [reply examples/inundation_qld.ipynb]: fixed I think

**Comment by emmaai** [comment]
Well done!!

**Review by emmaai — APPROVED** [review]

**Pull request merged**